platform/linux-generic/include/odp/api/plat/packet_inlines.h

```diff
@@ -423,6 +424,18 @@ _ODP_INLINE uint32_t odp_packet_buf_size(odp_packet_buf_t pkt_buf)
 			_odp_pool_get(pool, uint32_t, ext_head_offset);
 }
 
+_ODP_INLINE void *odp_packet_buf_head(odp_packet_buf_t pkt_buf)
+{
+	odp_pool_t pool = _odp_pkt_get(pkt_buf, odp_pool_t, pool);
+	const uint32_t head_offset = _odp_pool_get(pool, uint32_t, ext_head_offset);
+
+	/* Check that pool is external */
+	if (odp_unlikely(!head_offset))
+		return ODP_PACKET_BUF_INVALID;
```

Comment:
Should return NULL on failure, as return type is pointer (not odp_packet_buf_t).

Reply:
Fixed in V2.